tests/testem.json

```diff
@@ -1,6 +1,10 @@
 {
     "test_page": "tests/all-tests.html",
+    "skip": "Headless Chrome,IE,PhantomJS",
```

Comment:
Is it possible to skip Safari as well?  It consistently fails for me on Catalina with that "file open dialog" error.